tests/rpc_server.rs

```diff
@@ -111,7 +112,7 @@ async fn test_eth_get_block_by_number() {
         .expect("specified block not found");
 
     assert_header(&block.header, &hwp.header);
-    assert_eq!(block.size, None);
+    assert_eq!(block.size, Some(U256::from(37729)));
```

Comment:
Hmmm... this is for block `17510000`? Seems like there's a different size value on etherscan: https://etherscan.io/block/17510000

Reply:
After couple hours of debugging I figured out the problem.

The Transactions should be encoded with envelope (additional rlp header for non-legacy transactions). As far as I can tell, there is no easy way to do so with current code, so I'm leaving `size` as `None` for now (but I added comment for future fix).